.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] Great job adding the node modules to the gitignore file, but I see that you didn't put the test.md file inside the gitignore, so kindly add the  test.md file inside the gitignore as it is also a part of the requirement :+1: